src/ontology/lexmatch/README.md

```diff
@@ -3,48 +3,48 @@
 * mondo-only: Positive mappings in MONDO not caught by the lexical mapping pipeline
```

Comment:
Here's the answer . You're correct.

Reply:
Ok, but what does mondo-only + unmapped mean? I mean, the word "unmapped"?

Reply:
`unmapped` means mappings that are in one but not the other. For example: 

| subject_id   | subject_label              | object_id    | predicate_id          | object_label       | mapping_justification     | mapping_tool | confidence | subject_match_field | object_match_field | match_string |
|--------------|----------------------------|--------------|-----------------------|---------------------|---------------------------|--------------|------------|---------------------|--------------------|--------------|
| ID           | A oboInOwl:hasDbXref       | >A oboInOwl:source      |                       |                     |                           |              |            |                     |                    |              |
| MONDO:0000275 | obsolete monogenic disease | DOID:0050177 | MONDO:equivalentTo    | monogenic disease   | semapv:UnspecifiedMatching |              |            |                     |                    |              |

This mapping from `unmapped_doid_mondo.tsv` is in `mondo.sssom.tsv` but was not discovered in `lexmatch`. Hence `mondo-only`.